src/Library/demos/List View/main.blp

```diff
@@ -0,0 +1,88 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "List Model";
+  description: _("List models are a simple interface for ordered lists of GObject instances");
+  valign: start;
+
+  Adw.Clamp {
+    maximum-size: 240;
+    Box {
+      orientation: vertical;
+      spacing: 24;
+
+      LinkButton {
+        label: "Documentation";
+        uri: "https://gjs.guide/guides/gio/list-models.html";
+      }
+
+      Box {
+        halign: center;
+        styles ["linked"]
+        Button add {
+          label: _("Add Item");
+          tooltip-text: _("Add Item");
+        }
+
+        Button remove {
+          label: _("Remove Item");
+          tooltip-text: _("Remove Item");
+          }
+      }
+
+      Gtk.StackSwitcher {
+        stack: stack;
+        halign: center;
+      }
+
+      Gtk.Stack stack {
+        transition-type: crossfade;
+        vexpand: true;
+
+        Gtk.StackPage {
+          name: "listview";
+          title: _("List View");
+          child:  Frame {
+            valign: start;
+            ListView list_view {
+              enable-rubberband: true;
+              factory: BuilderListItemFactory {
+                template ListItem {
+                  child: Adw.ActionRow {
+                    title: bind template.item as <StringObject>.string;
+                  };
+                }
+              };
+            }
+          };
+        }
+
+        Gtk.StackPage {
+          name: "gridview";
+          title: _("Grid View");
+          child:  Frame {
+            valign: start;
+            GridView grid_view {
+              orientation: horizontal;
+              max-columns: 8;
+              enable-rubberband: true;
```

Comment:
Not sure why this doesn't work, but let's leave it for now and figure that out later.